docs/mlsys21/v100_rtx6000_throughputs.md.md

```diff
@@ -0,0 +1,193 @@
+
+
+### Amazon EC2 for V100
+
+If you have never used Amazon EC2 before, you should request an account from <https://aws.amazon.com/ec2> to get access to the GPU VMs
+
+After getting access, navigate to the "EC2 Dashboard" -> "Launch instance"  pane to create an VM with V100 GPUs.
+- The GPU instance we used for accessing V100 GPUs on Amazon EC2 is [`p3.2xlarge`](https://aws.amazon.com/ec2/instance-types/p3/).
+- The p3.2xlarge instance contains 8 vCPUs and 61 GB host memory. If you selected a larger instance with more GPUs, docker can limit the amount of host resource allocated per GPU.
+- We used `NVIDIA Deep Learning AMI v20.06.3`
+- The host resource contains 8 vCPUs, 61 GB memory, but docker instances will limit this.
+- We used a standard EBS boot disk with 100GB of storage capacity.
+
+__Note: please make sure to turn off your VM instances as soon as you finish the experiments, as the instances are quite costly__
+
+### Local Machine for RTX6000
+If you have a machine with at least one RTX6000, you need to configure the NVIDIA driver and CUDA toolkits properly. And `pytorch` much be able to run on this machine.
+
+## Steps
+
+
+
+### Prepare codebase
+
+First, clone the repo and navigate to the project
+
+```bash
+# clone the code base
+git clone https://github.com/UofT-EcoSystem/hfta.git
+cd hfta
+```
+
+### Acquiring NVIDIA Nsight Compute CLI and DCGM
+
+We require two installation files (`.deb`) for Nsight Compute and DCGM pre-downloaded to build the docker image.
+
+- Nsight Compute: version `3.1.72`, downloaded under `third_party/nsys/nsys_cli_2020.3.1.72.deb`
+- DCGM version: version `2.0.10`, downloaded under  `third_party/dcgm/datacenter-gpu-manager_2.0.10_amd64.deb`
+
+In order to download the `.deb` files, you need to register a NVIDIA developer account via: <https://developer.nvidia.com/login>, after that, you can download the .deb file:
+- NVIDIA Nsight System at: <https://developer.nvidia.com/nsight-compute>
+- NVIDIA DCGM at: <https://developer.nvidia.com/dcgm>
+
+
+### Download and launch docker image
+
+Follow the commands below to prepare and launch the docker image, this will take approximately 10 mins
+
+#### Build docker image
+```bash
+# build the image, select native1.6-cu10.2 for V100 and RTX6000 
+# this will take about 10 mins to complete
+bash docker/build.sh <the version of the image, e.g. native1.6-cu10.2>
+```
+
+#### Reuse prebuilt docker image
+If you do not wish to build the docker image from scratch, you can reuse the
+prebuilt docker image that we provide:
+```
+docker pull wangshangsam/hfta:mlsys21_native1.6-cu10.2
+docker tag wangshangsam/hfta:mlsys21_native1.6-cu10.2 hfta:dev
+```
+
+
+#### Launch a docker container
+```bash
+# launch the image
+# you will need to provide a placeholder mount point for the data directory
+# default is under ${HOME}/datasets
+ubuntu@ip-xxxxxxxx:~/hfta$ bash docker/launch.sh <optional: data directory mount point> <optional: image tag>
+
+=============
```

Comment:
The output below is actually not what you would see from an `native1.6-cu10.2` image (notice the `Various files include modifications (c) NVIDIA CORPORATION. All rights reserved.` here, which means NVIDIA-built NGC images). 

Just remove the output.